hail/python/hailtop/batch/docs/service.rst

```diff
@@ -79,6 +79,10 @@ machines with 100 GB of disk total. The costs are as follows:
 
 The sum of these costs is **$0.02170** per core per hour.
 
+Some jobs may be run on 4 core machines instead of 16 cores. In that case,
```

Comment:
Let's be a bit more specific in an attempt to head off people asking us questions:

> At any given moment as many as four cores of the cluster may come from a 4 core machine. If a job is 
> scheduled on this machine, ...